lua/starfall/libs_sh/builtins.lua

```diff
@@ -531,6 +531,10 @@ if SERVER then
 	-- @param ... printArgs Values to print. Colors before text will set the text color
 	function builtins_library.print(...)
 		local data, strlen, size = argsToChat(...)
+		if instance.player == SF.Superuser then
+			print("[SF] ".. ...)
+			return
+		end
```

Comment:
This should send to all players perhaps? Maybe sendPrintToPlayer can be edited to take nil for ply and use net.Broadcast if it's nil.

Reply:
It should only print on the server, wouldn't make much sense to send it to everyone

Reply:
All right, could you move it before the argesToChat call? Also I think it should be `print("[SF] ", ...)`

Reply:
or `print("[SF] ", unpack(data))`